components/CredentialDetailsViews.vue

```diff
@@ -239,6 +263,8 @@ export default {
       //  ]
       //
       if(!template || url) {
+        // TEMP FIX UNTIL PLAYGROUND DEPLOYS
+        url = url.replace('vcplayground.org', 'localhost:51443');
```

Comment:
So we definitely don't want this in here. There should be another way to do this sort of testing as well.

```suggestion
```

Reply:
100% - that was an accident! Updated [e2f9d3a](https://github.com/digitalbazaar/bedrock-vue-wallet/pull/98/commits/e2f9d3a9dcab9122657d4202cf21e3c8ec0222d5)